mscore/prefsdialog.cpp

```diff
@@ -1141,7 +1204,6 @@ void PreferenceDialog::apply()
             preferences.setPreference(PREF_IMPORT_STYLE_STYLEFILE, importStyleFile->text());
       else
             preferences.setPreference(PREF_IMPORT_STYLE_STYLEFILE, "");
-
       int ticks = MScore::division/4;
       switch(shortestNote->currentIndex()) {
```

Comment:
but add the missing space between `switch` and `(` here ;-)

Reply:
Will do :-)